Scripts/UtilityFunctions.gd

```diff
@@ -0,0 +1,13 @@
+class_name UtilityFunctions
+
+## UtilityFunctions class:
+## a class containing static functions for global use.
+
+## Gets all of the parents children (grand children, great grand children, ect)
+static func get_all_Children(root_node) -> Array[Node]:
```

Comment:
This implementation is fine for now, but later down the line we may want to replace/refactor it. No changes needed atm.

Reply:
do you mean becuse it is recursive?

Reply:
More so merging it with a community global functions script.